server.py

```diff
@@ -24,18 +24,28 @@
     allow_headers=['*']
 )
 
+class UserQuery(BaseModel):
+    query: str
+
 @app.get('/api')
 async def root():
     log.info("server running")
     log.info("server running")
     return {"message": "Server running"}
 
-@app.get('/api/evals')
-async def get_evals():
-    eval_table = evals.get_running_evals()
-    return {"message": eval_table}
 
-@app.post('/api/upload')
+
+
+
+
+# QUERY/CHAT ROUTES
+
+# api route for document uploads to add to document collection
+# TESTED, WORKING 7/29/24
+# required key/values for Postman:
+    # file => upload file via Postman
+    # filename => file name
+@app.post('/api/upload') 
 async def upload(file: UploadFile=File(...)):
```

Comment:
This function name is in conflict (will be shadowed) by the later file upload route - we may need to better organize route naming. Perhaps we could put all eval-specific routes under /api/evals

Reply:
renamed '/api/benchmark/upload' function to 'benchmark_upload'